integration/kubernetes/cleanup_bare_metal_env.sh

```diff
@@ -9,13 +9,14 @@ set -o nounset
 set -o pipefail
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
-iptables_cache="${SCRIPT_PATH}/iptables_cache"
+source "${SCRIPT_PATH}/../../lib/common.bash"
+
+iptables_cache="${KATA_TESTS_DATADIR}/iptables_cache"
```

Comment:
you should probably (have) merge this change (update) into the first commit, and have moved the KATA_TESTS_DATADIR commit to be the first commit :-)

Reply:
ok~~~~update soon. ;)